src/components/tagsInput/tagsInput.tsx

```diff
@@ -193,7 +182,7 @@ export class TagsInput extends React.Component<ITagsInputProps, ITagsInputState>
      * Gets tag ID (name) from a click event
      * @param event Click event
      */
-    private getTagIdFromClick(event): string {
+    private getTagIdFromClick = (event): string => {
```

Comment:
For learning: what's the difference here between `= () =>` and `() {`?

Reply:
@tbarlow12 - The fat arrow/lambda preserve the `this` context during callbacks and eliminates the need for us to use the `.bind()` syntax in the constructors.

Some other good info @ https://dmitripavlutin.com/6-ways-to-declare-javascript-functions/